libstb/secvar/backend/edk2-compat.c

```diff
@@ -183,8 +175,18 @@ static int edk2_compat_process(void)
 	}
 
 	if (rc == 0) {
-		clear_bank_list(&variable_bank);
-		variable_bank = staging_bank;
+		list_for_each(&staging_bank, node, link) {
+			tmp = find_secvar(node->var->key, node->var->key_len, &variable_bank);
```

Comment:
This is O(n^2). I know we are not concerned about optimization right now, but I don't think this is the way we should be operating our variable updating. Lot of indentation in this block as well.

Reply:
I think we should just figure out a `secvar_util.c` helper for copying a bank, so we don't have to reproduce this logic